CHANGELOG.md

```diff
@@ -7,6 +7,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.6.0] - 2020-06-12
+
+### Added
+
+- Indication of no-load and open lines in network display and single line diagrams
+
+### Changed
+
+- Misc. performance enhancements on the back-end side
```

Comment:
enhancments of the ?

Reply:
It seems to be [enhancements](https://dictionary.cambridge.org/fr/dictionnaire/anglais/enhancement)

Reply:
yes about enhancements (it was a typo) what i wanted is :
performance enhancements of the back-end side
enhancement of something